huaweicloud/services/gaussdb/data_source_huaweicloud_gaussdb_cassandra_flavors.go

```diff
@@ -101,9 +100,9 @@ func dataSourceCassandraFlavorsRead(_ context.Context, d *schema.ResourceData, m
 
 	filterFlavors, err := utils.FilterSliceWithField(allFlavors.Flavors, filter)
 	if err != nil {
-		return fmtp.DiagErrorf("filter Gaussdb cassandra flavors failed: %s", err)
+		return diag.Errorf("filter GaussDB cassandra flavors failed: %s", err)
 	}
-	logp.Printf("filter %d Gaussdb cassandra flavors from %d through options %v", len(filterFlavors), len(allFlavors.Flavors), filter)
+	log.Printf("filter %d GaussDB cassandra flavors from %d through options %v", len(filterFlavors), len(allFlavors.Flavors), filter)
```

Comment:
logging level is missing
```suggestion
	log.Printf("[DEBUG] filter %d GaussDB cassandra flavors from %d through options %v", len(filterFlavors), len(allFlavors.Flavors), filter)
```

Reply:
done